src/components/modal.css

```diff
@@ -22,6 +24,54 @@
     visibility: hidden;
   }
 
+  .modal {
```

Comment:
This shouldn't be nested. Even though they are meant to be used together, these are separate blocks. Nesting them like this makes it harder and require more specificity to override them